.github/workflows/ci.yml

```diff
@@ -361,23 +363,12 @@ jobs:
       - name: Upgrade ansible-core
         run: python3 -m pip install --upgrade ansible-core
 
-      - name: Download coverage artifacts A to H
+      - name: Download coverage artifacts
         uses: actions/download-artifact@v4
         with:
-          name: coverage-a-h
-          path: coverage
-
-      - name: Download coverage artifacts I to P
-        uses: actions/download-artifact@v4
-        with:
-          name: coverage-i-p
-          path: coverage
-
-      - name: Download coverage artifacts Z to Z
-        uses: actions/download-artifact@v4
-        with:
-          name: coverage-r-z0-9
```

Comment:
the fact that I successfully got a combined coverage artifact substantiates that this worked, somehow.

Reply:
Artifacts API v4 has `merge-multiple: true` which smashes the dirs in the `pattern:`-matching artifacts together into a common layout.